content/docs/1.6.0/advanced-resources/deploy/priority-class.md

```diff
@@ -7,10 +7,13 @@ The Priority Class setting can be used to set a higher priority on Longhorn work
 For more information on how pod priority works, refer to the [official Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/).
 
 # Setting Priority Class
+
 Longhorn system contains user deployed components (e.g, Longhorn manager, Longhorn driver, Longhorn UI) and system managed components (e.g, instance manager, engine image, CSI driver, etc.)
 You need to set Priority Class for both types of components. See more details below.
 
 ### Setting Priority Class During Longhorn Installation
+
+1. Longhorn will create and set the default priority class `longhorn-critical` if following actions are not taken.
```

Comment:
Shouldn't be numbered.  Actions to be taken should start with 1.

Reply:
Removed. Thanks